apps/dashboard/docker-entrypoint.sh

```diff
@@ -5,6 +5,10 @@ ENV_SCRIPT="<script>
   window._env_ = {
     VITE_API_HOSTNAME: '${VITE_API_HOSTNAME}',
     VITE_WEBSOCKET_HOSTNAME: '${VITE_WEBSOCKET_HOSTNAME}'
+    VITE_SELF_HOSTED_TOKEN: '${VITE_SELF_HOSTED_TOKEN}',
+    IS_CONTROLS_AUTOCOMPLETE_ENABLED: ${IS_CONTROLS_AUTOCOMPLETE_ENABLED:-false},
+    IS_WORKFLOW_NODE_PREVIEW_ENABLED: ${IS_WORKFLOW_NODE_PREVIEW_ENABLED:-false},
```

Comment:
@djabarovgeorge I could find rest of them in `apps/dashboard/src/config/index.ts`. How will code read these values from environment variable.